notebooks/WFC3/point_spread_function/hst_point_spread_function.ipynb

```diff
@@ -0,0 +1,1531 @@
+    "mean, median, std = sigma_clipped_stats(sci_driz, sigma=3.0)\n",
+    "daofind = DAOStarFinder(threshold=5, fwhm=3.0, roundhi=0.5)\n",
+    "mask = create_mask(sci_driz, cutout_size, 1823, 1096)\n",
+    "sources = daofind(sci_driz - median, mask=mask)\n",
+    "positions = np.transpose((sources['xcentroid'], sources['ycentroid']))\n",
+    "apertures_stellar = CircularAperture(positions, r=5.0)\n",
+    "apertures_annulus = CircularAnnulus(positions, r_in=9, r_out=12)\n",
+    "phot_table = aperture_photometry(sci_driz, apertures_stellar)\n",
+    "\n",
+    "# Calculate the sigma-clipped background levels and subtract that from the flux measured within the stellar aperture.\n",
+    "aperture_stats = ApertureStats(sci_driz, apertures_stellar, sigma_clip=None)\n",
+    "background_stats = ApertureStats(sci_driz, apertures_annulus, sigma_clip=SigmaClip(sigma=3.0, maxiters=10))\n",
+    "total_background = background_stats.median * aperture_stats.sum_aper_area.value # area of annulus.\n",
+    "flux_background_sub = aperture_stats.sum - total_background\n",
+    "phot_table['total_background'] = total_background\n",
+    "phot_table['flux_background_sub'] = flux_background_sub\n",
+    "\n",
+    "for col in phot_table.colnames:  \n",
+    "    if col not in ('id', 'npix'):\n",
+    "        phot_table[col].info.format = '%.2f'\n",
+    "print('\\n', phot_table, '\\n')\n",
+    "\n",
+    "plot_apertures(data=sci_driz, \n",
+    "               xcenter=1823, \n",
+    "               ycenter=1096, \n",
+    "               cutout_size=cutout_size, \n",
+    "               apertures_stellar=apertures_stellar, \n",
+    "               apertures_annulus=apertures_annulus)\n",
+    "\n",
+    "save_figure(figure, plot_dir+'fig9.pdf', True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4caca377-2d38-44e1-8f1b-30975c973f8c",
+   "metadata": {},
+   "source": [
+    "Finally, we use the stacked model from the drizzled image to perform a PSF fit and subtraction of the same four stars examined earlier. Overall, the model accurately encapsulates the total flux of each star, but some artifacts remain because drizzled images resample the observations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d6a19b3e-3584-4651-9b2c-cc9fd26db6dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "driz_stack_psf = FittableImageModel(driz_stack[0])\n",
+    "photometry, psfphot = calculate_photometry(sci_driz, sources, psf_model=driz_stack_psf)\n",
+    "resid = psfphot.make_residual_image(sci_driz, (17, 17))\n",
+    "figure = plot_psf_results(sci_driz, resid, 1823, 1096, cutout_size)\n",
+    "save_figure(figure, plot_dir+'fig10.pdf', True)\n",
+    "calculate_residuals(resid, sources)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7dc383b4-a3fa-45bb-b116-91049b317bd6",
+   "metadata": {},
+   "source": [
+    "Thank you for completing this tutorial."
```

Comment:
Annalisa noted that most of the information in the conclusions, such as pointing out that this can also be used with ACS, etc. really would be more appropriate in the introduction, so I've moved that information there. A formal conclusions section seemed a bit superfluous, although this could be a great spot to incorporate the summary you suggested.